contrib/macdeploy/gen-sdk

```diff
@@ -81,7 +81,7 @@ def run():
 
     print("Creating output .tar.gz file...")
     with out_sdktgz_path.open("wb") as fp:
-        with gzip.GzipFile(fileobj=fp, compresslevel=9, mtime=0) as gzf:
+        with gzip.GzipFile(fileobj=fp, mode='wb', compresslevel=9, mtime=0) as gzf:
```

Comment:
Why is this change necessary?

Reply:
This is a port from bitcoin commit 655d52a. I was made while attempting to upgrade macdeploy and its related script. Its not necessary in tapyrus.